services/github/github-contributors.service.js

```diff
@@ -7,10 +7,32 @@ const {
 } = require('../../lib/badge-data')
 const { metric } = require('../../lib/text-formatters')
 const {
+  documentation,
   checkErrorResponse: githubCheckErrorResponse,
 } = require('./github-helpers')
 
 module.exports = class GithubContributors extends LegacyService {
+  static get category() {
+    return 'other'
+  }
+
+  static get route() {
+    return {
+      base: 'github',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'GitHub contributors',
+        previewUrl: 'contributors/cdnjs/cdnjs',
```

Comment:
Should this one be

```
base: 'github/contributors',

previewUrl: 'cdnjs/cdnjs',
```

Reply:
Thinking ahead: that won't work when this is rewritten, because there's `contributors-anon` in the pattern, and BaseService wants a slash between `base` and `pattern`.

(There are a bunch of others like this.)